src/main/java/com/telefonica/iot/cygnus/sinks/OrionHDFSSink.java

```diff
@@ -263,6 +264,18 @@ public void configure(Context context) {
                     + "properly work!");
         } // if else
         
+        String defaultCsvSeparator = context.getString("default_csv_separator");
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/93781aa1f586022ef98d1f1612a7c737add79f30